spec/text-editor-component-spec.coffee

```diff
@@ -1352,17 +1352,18 @@ describe "TextEditorComponent", ->
         expect(overlay.style.left).toBe position.left + gutterWidth + 'px'
         expect(overlay.style.top).toBe position.top + editor.getLineHeightInPixels() + 'px'
 
-        editor.insertText('a')
-        nextAnimationFrame()
+        if process.platform isnt "linux" # see TextEditorComponent#measureWindowSize
```

Comment:
Are the asserts before this line still worth running on linux?

If not, you can mark this spec as not running on linux by changing the `it` to be:

```coffee
it "slides horizontally left when near the right edge on #win32 and #darwin", ->
```